src/storm-pars/settings/modules/SamplingSettings.cpp

```diff
@@ -25,6 +26,7 @@ SamplingSettings::SamplingSettings() : ModuleSettings(moduleName) {
                                                    "Sets whether it can be assumed that the samples are graph-preserving.")
                         .build());
     this->addOption(storm::settings::OptionBuilder(moduleName, sampleExactOptionName, false, "Sets whether to sample using exact arithmetic.").build());
+    this->addOption(storm::settings::OptionBuilder(moduleName, sampleDerivativeOptionName, false, "Sets whether to sample the derivatives instead..").build());
```

Comment:
Don't we always solve the system, also when computing derivatives? So is the "instead" maybe confusing / unnecessary?

Reply:
ok will change it!